node/tests/swamp/config.go

```diff
@@ -0,0 +1,28 @@
+package swamp
+
+import (
+	"time"
+
+	"github.com/tendermint/tendermint/abci/types"
+	tn "github.com/tendermint/tendermint/config"
+
+	"github.com/celestiaorg/celestia-node/core"
+)
+
+// Config struct represents a set of pre-requisite attributes from the test scenario
+type Config struct {
+	App     types.Application
```

Comment:
Semantically, it is strange to have a whole App under the config, but it might be ok.

Reply:
tbh, I haven't came up with a more neat idea, yet. I propose we leave this as is and come back as we will start playing with core/app side more that just consensus and kvStores.